Noti/PushManager.swift

```diff
@@ -460,6 +422,118 @@ class PushManager: NSObject, WebSocketDelegate, NSUserNotificationCenterDelegate
         
     }
     
+    // Given a push JSON object from Pushbullet create a notification
+    // Used in mirror and subtipe:push
+    internal func createNotification(msg : JSON) -> NSUserNotification{
```

Comment:
FYI, [when providing a underscore in front of the parameter, it can be omitted.](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Functions.html#//apple_ref/doc/uid/TP40014097-CH10-ID526)
Thus, you won't need to call the function as `createNotification(msg: data)` but can instead just `createNotification(data)`.
Parameter labels are great for reading what exactly the function is expecting, but in this case it's pretty clear what the function is expecting: a message object to create a notification from.